### PR TITLE
Added initial run of typescript generation.

### PR DESCRIPTION
**Pull request by david-driscoll**
Trying to solve for #180  I know I need to make changes to build.sh, I'll see if I can get my ubuntu vm setup for building and such.

### CODE REVIEW

File: src/OmniSharp.TypeScriptGeneration/OmniSharp.TypeScriptGeneration.kproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (nosami) [inline src/OmniSharp.TypeScriptGeneration/OmniSharp.TypeScriptGeneration.kproj]: eurgh... you used VS here

Comment (david-driscoll) [inline src/OmniSharp.TypeScriptGeneration/OmniSharp.TypeScriptGeneration.kproj]: :sheep:

**Comment by nosami** [comment]
I'm starting to think that this code belongs with the omnisharp-server-roslyn-binaries repo.

**Comment by nosami** [comment]
Would be interested to hear what @davidfowl and @jrieken think about this.

**Comment by jrieken** [comment]
:+1: for using the ```omnisharp-server-roslyn-binaries```-repo for this. that's the thing you consume when integrating omnisharp into another editor.

File: src/OmniSharp.TypeScriptGeneration/Program.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+using OmniSharp.Stdio;
+using OmniSharp.Stdio.Protocol;
+using TypeLite;
+using TypeLite.TsModels;
+
+namespace OmniSharp.TypeScriptGeneration
+{
+    public class Program
+    {
+
```
Comment (davidfowl) [inline src/OmniSharp.TypeScriptGeneration/Program.cs]: empty space

File: src/OmniSharp.TypeScriptGeneration/Program.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        public void Main(string[] args)
+        {
+            var path = string.Empty;
+            if (args.Length == 1)
+            {
+                path = args[0];
+            }
+
+
```
Comment (davidfowl) [inline src/OmniSharp.TypeScriptGeneration/Program.cs]: Empty space

File: src/OmniSharp.TypeScriptGeneration/Program.cs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            var fluent = TypeScript.Definitions();
+            fluent.ScriptGenerator.IndentationString = "    ";
+
+            fluent.WithMemberTypeFormatter(TsFluentFormatters.FormatPropertyType);
+            fluent.WithMemberFormatter(TsFluentFormatters.FormatPropertyName);
+            //definitions.WithTypeFormatter(IgnoreInvalidTypes);
```
Comment (davidfowl) [inline src/OmniSharp.TypeScriptGeneration/Program.cs]: dead code

**Comment by david-driscoll** [comment]
Moved this over to the binaries repo.

**Pull request closed without merging**